Source/santad/SNTExecutionController.m

```diff
@@ -365,44 +373,36 @@ - (void)printMessage:(NSString *)msg toTTYForPID:(pid_t)pid {
     return;
   }
 
-  NSString *devPath = [NSString stringWithFormat:@"/dev/%s", devname(taskInfo.e_tdev, S_IFCHR)];
-  int fd = open(devPath.UTF8String, O_WRONLY | O_NOCTTY);
-  @try {
-    NSFileHandle *fh = [[NSFileHandle alloc] initWithFileDescriptor:fd closeOnDealloc:YES];
-    [fh writeData:[msg dataUsingEncoding:NSUTF8StringEncoding]];
-  } @catch (NSException *) { /* do nothing */ }
+  char devPath[32] = "/dev/";
```

Comment:
32 seems arbitrary. Either use `PATH_MAX` or go the other way and count the max tty sessions. `kern.tty.ptmx_max` seems to take a max of 999. So the max "/dev/tty999" is 12 bytes with null termination.